app/views/registers/show.html.haml

```diff
@@ -2,142 +2,139 @@
 - @page_description = "#{[@register.meta_description, @register.register_description].find(&:present?)}"
 - content_for :body_classes, 'register-show'
 
-%main#content.container{role:'main'}
-  - if @register.register_phase != 'Beta'
-    .grid-row
-      .column-two-thirds
-        .panel.panel-border-wide
-          %span.phase-banner In progress
-          %p This data is in progress and it’s not ready for use.
-
-  .register-meta-data
-    .grid-row
-      .column-two-thirds
-        %h1{class: "govuk-heading-l govuk-!-margin-top-7 govuk-!-margin-bottom-1"}
+.govuk-width-container
+  %main#content.govuk-main-wrapper{role:'main', class: 'govuk-!-padding-top-0'}
+    .govuk-grid-row{class: 'govuk-!-margin-top-5'}
+      - if @register.register_phase != 'Beta'
+        .govuk-grid-column-full
+          .govuk-inset-text
+            %strong.govuk-tag In progress
+            %p This data is in progress and it’s not ready for use.
+      .govuk-grid-column-two-thirds{class: 'govuk-!-margin-top-2'}
+        %h1{class: "govuk-heading-l govuk-!-margin-bottom-1"}
           #{@register.register_name} register
         %p.govuk-body-l= @register.register_description
-        - unless @register.is_empty?
-          %p.font-xsmall Last updated: #{link_to formatted_date(@register.register_last_updated), register_entries_path(@register.slug)}
-
-      .column-one-third
+        %dl.govuk-metadata
+          - if @register.authority.present?
+            %dt.register-metadata__term Organisation:
+            %dd.register-metadata__definition= link_to @register&.authority&.name, authority_path(@register.authority.government_organisation_key)
+          - if @register&.category_id&.present? or @register.register_phase != 'Beta'
```

Comment:
we could move this logic into the model e.g.
```
 def show_category_link
   register_phase == 'Beta' && category.present?
 end
```

Reply:
Sounds good; now done.